sdk/loadtesting/azure-developer-loadtesting/azure/developer/loadtesting/_generated/operations/_patch.py

```diff
@@ -70,7 +70,7 @@ def __init__(self, interval=5) -> None:
         self._initial_response = None
         self._polling_interval = interval
         self._status = None
-        self._termination_statuses = ["VALIDATION_SUCCESS", "VALIDATION_FAILED", "VALIDATION_NOT_REQUIRED"]
+        self._termination_statuses = ["VALIDATION_SUCCESS", "VALIDATION_FAILURE", "NOT_VALIDATED", "VALIDATION_NOT_REQUIRED"]
```

Comment:
do we not need this at this place as well? https://github.com/Azure/azure-sdk-for-python/blob/b83fdce650f0a42b0e18cf437fe52525a24d534d/sdk/loadtesting/azure-developer-loadtesting/azure/developer/loadtesting/_generated/aio/operations/_patch.py#L66